pages/index.js

```diff
@@ -229,6 +261,33 @@ const Home = (props) => {
 
         <div className={styles.contactUs} id="contact">
           <FadeInSection>
+          <script>
```

Comment:
Shall we have the statistics displayed near the top of the screen, below About Us? Feel free to use these for inspiration:

![image](https://user-images.githubusercontent.com/50544486/134760003-9f9d2c72-bb6e-4787-ab1d-af6530075f35.png)

![image](https://user-images.githubusercontent.com/50544486/134760009-82ee915a-4989-4539-98d8-3f40baabb7a9.png)